akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardRegion.scala

```diff
@@ -367,7 +362,13 @@ object ShardRegion {
    *
    * If gathering the shard information times out the set of shards will be empty.
    */
-  @SerialVersionUID(1L) final case class CurrentShardRegionState(shards: Set[ShardState]) {
+  @SerialVersionUID(1L) final case class CurrentShardRegionState(shards: Set[ShardState], failed: Set[ShardId])
```

Comment:
Sure, I did start this change out by doing that, then reverted to the original case class thinking because it was not yet in the serializer and sent over the wire it may not need it. I"ll make the change, thanks.

Reply:
I figured out why mima missed this: the fix at https://github.com/lightbend/mima/pull/468/files#diff-1b17b4fbc8ceecfa206a29f9a469ff3aR21 is not yet released.

Reply:
verified with a snapshot from mima master, there it does catch this problem and successfully verifies the latest version of this branch is OK :+1:

Reply:
Thanks!